stm32-data-gen/src/chips.rs

```diff
@@ -536,6 +536,8 @@ impl PeriMatcher {
             (".*:CRYP:cryp1_v1_0.*", ("cryp", "v1", "CRYP")),
             (".*:CRYP:cryp1_v2_0.*", ("cryp", "v2", "CRYP")),
             (".*:CRYP:cryp1_v2_2.*", ("cryp", "v2", "CRYP")),
+            ("STM32G0.*1.*:.*:COMP:.*", ("comp", "v1", "COMP")),
```

Comment:
If you mean "filter G0x1", then it’s better to use "STM32G0?1.*"?

Reply:
Haha yep I do mean that, I am *not* good at regex lol, just tried copying what was above.

Reply:
Hmm, didn't seem to work when I pasted your suggestion in. I think i'll leave it alone for now.

Reply:
Oh I think you meant `.` instead of `?`, will update, thanks.

Reply:
Oh sorry I made a mistake, it’s "STM32G0.1"